gem/lib/earlgrey/configure_earlgrey_pods.rb

```diff
@@ -0,0 +1,388 @@
+#
+#  Copyright 2016 Google Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+# global export for unqualified use in Pods file
+def configure_for_earlgrey(*args)
+  EarlGrey.configure_for_earlgrey(*args)
+end
+
+module EarlGrey
+  class << self
+    attr_accessor :swift, :carthage
+    attr_reader :project_name, :test_target, :test_target_name, :scheme_file, :user_project
+
+    # Returns path to Xcode file, prepending current working dir if necessary.
+    # @param xcode_file [String] xcode file path
+    # @param ext [String] xcode file extension
+    # @return [String] path to Xcode file
+    def path_for(xcode_file, ext)
+      return xcode_file if File.exist? xcode_file
+
+      path = File.join(Dir.pwd, File.basename(xcode_file, '.*') + ext)
+      path ? path : nil
+    end
+
+    # Strips excessive spaces from a string
```

Comment:
Update comment to mention that spaces are striped from individual lines (unless thats not what you intended to do).